lambda-pure.zsh

```diff
@@ -151,7 +151,10 @@ prompt_pure_preprompt_render() {
 	preprompt+="%F{cyan}${prompt_pure_cmd_exec_time}%f"
 
 	# NodeJS version
-	local rpreprompt="%F{green}⬢ ${prompt_pure_node_version}%f"
+	local rpreprompt
+	if ! (( ${PURE_NODE_DISABLED} )); then
```

Comment:
I would rather have it the other way around, i.e. variable called `PURE_NODE_ENABLED` which is true by default. I don't like negative variables that you set to true in order to disable something ;)

Reply:
That makes sense - I changed it.